### PR TITLE
Create KOTO.yaml

### PR DESCRIPTION
**Pull request by ppaschos**
Adding KOTO project in topology

### CODE REVIEW

**Comment by ppaschos** [comment]
I think there was an error in the university: It should be the "The University of Chicago" instead of just "University of Chicago"

File: projects/KOTO.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Description is a human-readable description of what the project is about
+# it can be multiple lines as long as each line is indented.
+Description:
```
Comment (mwestphall) [inline projects/KOTO.yaml:3]:
```suggestion
Description: KOTO Collaboration OSG project on ap23
```

File: projects/KOTO.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+KOTO Collaboration OSG project on ap23
```
Comment (mwestphall) [inline projects/KOTO.yaml:4]:
```suggestion

```

File: projects/KOTO.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  ### one of the file names in the virtual-organizations/ directory (minus the .yaml extension)
+  VirtualOrganization:
+    Name: OSG
+
+
+### Uncomment the ResourceAllocations block if this project can make use of
+### one or more XRAC (XSEDE) or other HPC allocations.
+###
+### Contact the Software Team for assistance in filling in these values.
+# ResourceAllocations:
```
Comment (mwestphall) [inline projects/KOTO.yaml:39]:
```suggestion
ResourceAllocations:
```

File: projects/KOTO.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+### Uncomment the ResourceAllocations block if this project can make use of
+### one or more XRAC (XSEDE) or other HPC allocations.
+###
+### Contact the Software Team for assistance in filling in these values.
+ResourceAllocations:
+
+##### Type is how the allocation was obtained: "XRAC" for allocations through
+##### XSEDE or "Other" for allocations directly through the site.
+#   - Type: Other
```
Comment (mwestphall) [inline projects/KOTO.yaml:43]:
```suggestion
   - Type: Other
```

**Review by mwestphall — APPROVED** [review]

**Pull request merged**